Scoop/Controller/MatchesViewController.swift

```diff
@@ -8,11 +8,62 @@
 import UIKit
 
 class MatchesViewController: UIViewController {
-
+    // MARK: Views
+    private let tableView = UITableView()
+    
+    // MARK: Identifiers
+    private let homeCellIdenitifer = "HomeCell"
+    
+    var matchedRides: [Ride] = []
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
         navigationItem.title = "Best Matches"
+        
+        setupTableView()
+    }
+    
+    init(rides: [Ride]) {
+        super.init(nibName: nil, bundle: nil)
+        self.matchedRides = rides
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setupTableView() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.contentInset = UIEdgeInsets(top: 20, left: 0, bottom: 0, right: 0)
+        tableView.backgroundColor = .white
+        tableView.estimatedRowHeight = UITableView.automaticDimension
+        tableView.separatorStyle = .none
+        tableView.register(HomeTableViewCell.self, forCellReuseIdentifier: homeCellIdenitifer)
+        view.addSubview(tableView)
+        
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide)
+            make.leading.trailing.equalToSuperview()
+            make.bottom.equalTo(view.safeAreaLayoutGuide)
+        }
+    }
+}
+
+extension MatchesViewController: UITableViewDelegate {
```

Comment:
If we don't use any delegate functions can we just delete this extension and delete `tableView.delegate = self`